requirements.txt

```diff
@@ -49,7 +49,7 @@ simplejson==3.16.0
 six==1.11.0
 Sphinx==1.2.3
 sphinx-rtd-theme===0.1.6-ga3
-SQLAlchemy==1.0.14
+SQLAlchemy==1.3.0
```

Comment:
does it make sense to update directly to the latest minor release, 1.3.4?

Reply:
yes